### ISSUE TITLE
Candidate changes for 0.6.2 release

### ISSUE THREAD

**Issue by matthewhanson** [radiantearth/stac-spec#388]
A 0.6.2 release should only include minor non-breaking changes, such as fixing typos or other errors.

Here is the comparison of master to dev:
https://github.com/radiantearth/stac-spec/compare/master...dev

Here are list of candidate commits that should be included in a 0.6.2 release

- Fixed example (change band names to id): https://github.com/radiantearth/stac-spec/commit/645dd1ff3087de97adeb08c679eef1f2d1590709

- Add missing example links to commons extension (examples were there, links from README were not): https://github.com/radiantearth/stac-spec/commit/0f92843a22f6db409a9b1836b78b621f76d80a44

- Add requirement for a schema in extensions: https://github.com/radiantearth/stac-spec/commit/016b5c86a5468b48934c4b14ff32919f4e00a3c6

- typo fix in bbox description: https://github.com/radiantearth/stac-spec/commit/860fb110c63984670b5678b5e583974f9922fab7

- replace id with $id in JSON schema: https://github.com/radiantearth/stac-spec/commit/71ab946ddc2fb329e37f7276ea2701a1786bd5e2

- Fix typo in Item spec bbox description: https://github.com/radiantearth/stac-spec/commit/c389de6a818c97803ae51798a207f6091ee3e471

**Comment by matthewhanson** [comment]
Also we should include adding of required properties to the JSON schema

https://github.com/radiantearth/stac-spec/pull/379

When a new PR to dev is created and merged, cc @mojodna

**Comment by m-mohr** [comment]
The list above may be a little outdated, see #390.

**Comment by matthewhanson** [comment]
I'll update list when https://github.com/radiantearth/stac-spec/pull/390 is merged and will review changes between master and dev

**Comment by m-mohr** [comment]
Why don't we include well-discussed extensions with multiple examples and a schema in minor releases? I think it would be valuable to get them out as soon as possible and new extensions are usually fully backward-compatible and non-breaking, so it would be covered by SemVer. An example would be the SAR extension (after #393 got merged). It has a schema, two examples and was 6 organizations contributed and even more reviewed it. I'd vote to include it in 0.6.2.
Point Cloud is missing a schema so that would need to be added before releasing it, see #391. 
cc @cholmes

**Comment by m-mohr** [comment]
#390 is merged now.

**Comment by matthewhanson** [comment]
Im fine with including extensions if they get merged in time.

The #390 merge doesn't change the above list, the only differences were version numbers (0.6.0 to 0.6.1).

Updated top post with commits for the merged #379 and #397

**Comment by m-mohr** [comment]
Thanks. #398 could be a candidate, too. But not sure about this one as it actually restricts the schema more. We would need to decide whether that is a mistake in the schema and backed by the written spec or not.

**Comment by cholmes** [comment]
I'm +1 on #398 - the point of having a version number is to make it useful.

**Comment by m-mohr** [comment]
#398 got merged and should be included (the version number needs to be changed, of course).

**Comment by m-mohr** [comment]
Candidate: #402

**Comment by m-mohr** [comment]
I quickly had a look at https://github.com/radiantearth/stac-spec/compare/master...dev and to me it seems dev is actually the 0.6.2, I don't see any breaking changes. We need to adopt version numbers and may wait for some more changes to be merged (e.g. #402, my PRs on units, crs, trs in the next days).

**Comment by cholmes** [comment]
+1 - would also be good to get #361 merged. But don't want to hold the release.

If there's not cherry picking needed I can help with more of the release. We'll just merge dev to master and adjust the version numbers, right? 

I'm close to done with my day, but I'll block aside a couple hours tomorrow to work on release. @matthewhanson - let me know if you're able to make progress and pass off, and no worries if you can't do stuff till friday, I'll go as far as I can.

**Comment by m-mohr** [comment]
Sorry, I had some back and forth with myself on the data cube extension, but didn't clearly articulated that it needs another iteration. As you worked hard tonight and merged #361, I had to open #409. Would be great if we could get that in, too.

Chris, I don't think cherry-picking is needed. Scrolling through the files, I don't find any breaking changes, but better double check: https://github.com/radiantearth/stac-spec/compare/master...dev

If you also don't find any breaking changes, we can probably go for a release. Work items I have in mind:
- [ ] update the version numbers
- [ ] update the CHANGELOG
- [ ] update the referenced API documents on SwaggerHub
- [ ] merge dev to master as 0.6.2.
- [ ] make a release
- [ ] announce the new version and the new extensions?

**Comment by m-mohr** [comment]
Has been released.
